taxcalc/functions.py

```diff
@@ -800,14 +811,26 @@ def TaxGains(e00650, c04800, e01000, c23650, e23250, e01100, e58990,
 # TODO should we return c00650 instead of e00650?? Would need to change tests
 
 
+@iterate_jit(nopython=True)
+def MUI(c00100, NIIT_thd, MARS, e00300, e00600, c01000, e02000, NIIT_trt,
+        NIIT):
+    # Additional Medicare tax on unearned Income
+    if c00100 > NIIT_thd[MARS - 1]:
+        NIIT = NIIT_trt * min(e00300 + e00600 + max(0, c01000) +
```

Comment:
I don't think this relocation is right. See line 62 of 1040. NIIT is added directly to othertax.